conductr_cli/sandbox_proxy.py

```diff
@@ -12,7 +12,7 @@
 
 HAPROXY_CFG_DIR = '{}/haproxy'.format(DEFAULT_SANDBOX_PROXY_DIR)
 HAPROXY_CFG_PATH = '{}/haproxy.cfg'.format(HAPROXY_CFG_DIR)
-DEFAULT_PROXY_PORTS = [80, 443, 9000]  # These are the ports which will be opened by default to the proxy.
+DEFAULT_PROXY_PORTS = [80, 443]  # These are the ports which will be opened by default to the proxy.
```

Comment:
This is remove in lieu of the existing `--bundle-http-port` default argument:

https://github.com/typesafehub/conductr-cli/pull/302/files#diff-6089c6531dbfde85064554aba3e7de40R153

So the value supplied by `--bundle-http-port` is passed into the `sandbox_proxy` to be exposed when starting the docker container.